### ISSUE TITLE
CMake error on linux

### ISSUE THREAD

**Issue by ghost** [Cmdu76/SDLBgfxTest#1]
I simply coned the repository and did: `./GenerateProject.sh`

```
[...]
CMake Error in bgfx/CMakeLists.txt:
Target "bgfx" INTERFACE_INCLUDE_DIRECTORIES property contains path:
    "/home/<user's account>/projects/SDLBgfxTest/bgfx/bgfx/3rdparty"
  which is prefixed in the source directory.
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by Caerind** [comment]
Yeah I modified something on master that I shouldn't have ^^
I will revert it as soon as possible

**Comment by ghost** [comment]
Have been fixed by commit 92c05ed1f007dcb659b03abcd6700bb4fa16bb55.
